docs/users-guide/10-pulses.md

```diff
@@ -1,4 +1,8 @@
-## Sharing updates with pulses
+## Dashboard subscriptions will replace Pulses in a future release
+
+[Dashboard subscriptions](dashboard-subscriptions) will be replacing Pulses in a future release of Metabase. Dashboard subscriptions allow you to send the contents of a dashboard on a schedule, without having to manually add each question (which Pulses require you to do). When we remove Pulses, we'll provide help for migrating your remaining Pulses to dashboard subscriptions. So if you're thinking of setting up a Pulse, try out a dashboard subscription instead.
```

Comment:
It seems a little odd that "Pulse" is capitalized and "dashboard subscription" is not.